USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -70,9 +125,24 @@ private string RenderMarker(Marker input)
                         output.Append(RenderMarker(marker));
                     }
                     output.AppendLine("</div>");
+                    output.AppendLine(RenderFootnotes());
+                    // Page breaks after each chapter
+                    output.AppendLine($"{(separateChapters ? "<br class=\"pagebreak\"></br>" : "")}");
+
                     break;
                 case VMarker vMarker:
                     output.AppendLine($"<span class=\"verse\">");
+
+                    //Interpret VPMarker
+                    switch (input.Contents[0])

```

Comment:
I would actually implement this differently. I would actually implement this as a property on the vMarker itself since it will be used in pretty much all renderers.

Reply:
The USFMParser splits up the USFM markers with the regex pattern \\([a-z0-9*]+)([^\\]*) 

When the VMarker hits Preprocess(), the only input is \v 1 . It doesn't view \vp 1a \vp* yet. 
I'm not sure how I can update the property as it parses the document.